### PR TITLE
Updates instructions for contributing and release instructions

### PR DESCRIPTION
**Pull request by moonso**
This PR updates CONTRIBUTING.md and RELEASE.md

**Review:**
- [x] code approved by CR

### CODE REVIEW

**Comment by moonso** [comment]
Super small @northwestwitch

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,30 +1,42 @@
 # Contributing to Scout
 
-Hi, nice to hear that you wan't to contribute to Scout! Please have a quick read through this text first. Contributing could be anything from questions and bug reports (issues), updating the documentation to fixing features.
+Hi, nice to hear that you wan't to contribute to Scout! Please have a quick read through this text
```
Comment (northwestwitch) [inline CONTRIBUTING.md]: that you want, without apostrophe

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,30 +1,42 @@
...
 
-Hi, nice to hear that you wan't to contribute to Scout! Please have a quick read through this text first. Contributing could be anything from questions and bug reports (issues), updating the documentation to fixing features.
+Hi, nice to hear that you wan't to contribute to Scout! Please have a quick read through this text 
+first. Contributing could be anything from questions and bug reports (issues), updating the 
+documentation to fixing features.
 
 ## Questions and Bugs
 
-First try to search the issues with some keywords to check if it has been touched before or if there is an ongoing discussion.
+First try to search the issues with some keywords to check if it has been touched before or if
```
Comment (northwestwitch) [inline CONTRIBUTING.md]: Before submitting a new bug or question try to look into the existing issues (open or closed). Your problem could already be solved or the object of an ongoing discussion.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,30 +1,42 @@
...
 
 ## Questions and Bugs
 
-First try to search the issues with some keywords to check if it has been touched before or if there is an ongoing discussion.
+First try to search the issues with some keywords to check if it has been touched before or if 
+there is an ongoing discussion.
 
 If you have found a new topic open a issue and describe as detailed as possible your question.
-For bug reports try to include error messages if there are any.
+For bug reports include error messages if there are any.
```
Comment (northwestwitch) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,30 +1,42 @@
...
 
 ## Contributor Guidelines
 
 ### Scout Git branching strategy
 
-Scout development is organized on a flexible Git ["Release Flow"] (https://www.nebbiatech.com/2019/03/15/git-branching-strategies-which-one-should-i-pick/) branching system. Our workflow is similar to the [GitHub Flow system](https://guides.github.com/introduction/flow/), but it differs from it for the fact we don't deploy master to production immediately after a pull request is merged. Instead we plan each new release of the software together with production team at our Facility. Before releasing a new Scout version we create a release branch and a [GitHub Release with a tag](https://github.blog/2013-07-02-release-your-software/). This tag points to the head branch of the repository.
-
-The easiest way to create a pull request to Scout would be to clone the repository and create your own feature branch, based on an updated master branch.
+Scout development is organised on a flexible Git ["Release Flow"][release_flow] branching system.
+This more or less means that we make releases in release brances, these are stable versions of Scout.
```
Comment (northwestwitch) [inline CONTRIBUTING.md]: we make releases in release brancHes, which correspond to stable versions of Scout

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,30 +1,42 @@
...
+Whenever a feature is added a branch is created from current master, when ready open a pull request
```
Comment (northwestwitch) [inline CONTRIBUTING.md]: In order to add a new feature create a branch from the current master and follow the instructions below:

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,30 +1,42 @@
...
 
 ### Pull requests
 
-1. We have included an automatic pull request template to help you defining the proposed changes/fixes and how to test the code.
-1. If the pull request add functionality the docs should be updated.
-1. Don't forget to update the CHANGELOG.md document.
+The easiest way to create a pull request to Scout would be to clone the repository and create your 
+own feature branch, based on an updated master branch.
+
+1. Please try to name your PR with dashes as separator like `adds-supertrack-to-igv`
```
Comment (northwestwitch) [inline CONTRIBUTING.md]: I use underscores and nobody said anything for 2 years

Comment (moonso) [reply CONTRIBUTING.md]: Haha I know...

Comment (moonso) [reply CONTRIBUTING.md]: It's not a big deal, just that we did like that from the beginning

**Comment by moonso** [comment]
I fixed according to your suggestions @northwestwitch

**Comment by moonso** [comment]
Done and done...

**Comment by dnil** [comment]
If you have time, the release instructions for updating gh-pages seems off by a little. It might have been a slip on my side, but since I didn't quite know how to use GitBook I really tried to follow the instructions..

**Comment by moonso** [comment]
Aha I will have a look at that

**Pull request merged**